src/commands/lint/index.ts

```diff
@@ -12,14 +12,18 @@ interface LintResult {
   errors: boolean
 }
 
+const excludeFolders = ['.git', '.github', '.vscode', 'node_modules']
```

Comment:
should we include `sasjsbuild` here also?  and `sasjsresults` ?

Am a bit confused as wouldn't expect .sas programs to be in the .git / .github / .vscode folders anyway, but I guess it can be better performance wise to exclude them

Reply:
fixed